vault/core.go

```diff
@@ -2500,8 +2500,8 @@ func (c *Core) preSeal() error {
 	if err := c.stopExpiration(); err != nil {
 		result = multierror.Append(result, fmt.Errorf("error stopping expiration: %w", err))
 	}
-	if err := c.sealManager.Reset(context.Background()); err != nil {
-		result = multierror.Append(result, fmt.Errorf("error reseting seal manager: %w", err))
+	if err := c.sealAllNamespaces(context.Background()); err != nil {
```

Comment:
@KrzysztofKornalewski-Reply @satoqz @wslabosz-reply I think this goes to my discussion with Wojciech yesterday on the horizontal scalability WG (and before on Zulip), but do we want to seal all namespaces at `preSeal(...)` or should this be called in place of `c.barrier.Seal()`? IMHO it needs to be the latter to allow proper standby->active changeover, otherwise we lose all Shamir's keys. 

Regardless, not really relevant to this PR, as it is not the focus of it.